m2Final.js

```diff
@@ -0,0 +1,58 @@
+const select = document.querySelector("#movie-titles");
+const chosenTitle = document.querySelector("#selected-title");
+const chosenRelease = document.querySelector("#release-yr");
+const chosenDescription = document.querySelector("#film-description");
+const reviewList = document.querySelector("#review-list");
+const reviewSubmission = document.querySelector("#review-submit");
+
+const createOptions = async () => {
+  try {
+    const option = document.createElement("option");
+    option.textContent = "";
+    select.appendChild(option);
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films/");
+    res.data.forEach((movie) => {
+      const option = document.createElement("option");
+      option.textContent = movie.title;
```

Comment:
we could of used the `value` option property to store the id of the current movie
`option.value = film.id`  to then later be used to get that single film